From 32a05d8e97d3de01cf6c5d8f962d2e1c14086840 Mon Sep 17 00:00:00 2001
From: Tejas Patel <tejas.patel1@einfochips.com>
Date: Thu, 11 Nov 2021 17:20:46 +0530
Subject: [PATCH 1/2] meta-oe: recipes-graphics: tesseract: Update git protocol
 and branch to use

Signed-off-by: Tejas Patel <tejas.patel1@einfochips.com>

diff --git a/meta-oe/recipes-graphics/tesseract/tesseract-lang_4.1.0.bb b/meta-oe/recipes-graphics/tesseract/tesseract-lang_4.1.0.bb
index fd2973bc2..f4e071c69 100644
--- a/meta-oe/recipes-graphics/tesseract/tesseract-lang_4.1.0.bb
+++ b/meta-oe/recipes-graphics/tesseract/tesseract-lang_4.1.0.bb
@@ -4,7 +4,7 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
 
 SRCREV = "4767ea922bcc460e70b87b1d303ebdfed0897da8"
-SRC_URI = "git://github.com/tesseract-ocr/tessdata.git"
+SRC_URI = "git://github.com/tesseract-ocr/tessdata.git;branch=master;protocol=https"
 
 S = "${WORKDIR}/git"
 
diff --git a/meta-oe/recipes-graphics/tesseract/tesseract_4.1.1.bb b/meta-oe/recipes-graphics/tesseract/tesseract_4.1.1.bb
index 3b5032f06..49ad4513d 100644
--- a/meta-oe/recipes-graphics/tesseract/tesseract_4.1.1.bb
+++ b/meta-oe/recipes-graphics/tesseract/tesseract_4.1.1.bb
@@ -7,7 +7,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
 
 BRANCH = "4.1"
 SRCREV = "f4ef2f2050f4c25b28bdbf0063b7d2eb30f41cf7"
-SRC_URI = "git://github.com/${BPN}-ocr/${BPN}.git;branch=${BRANCH} \
+SRC_URI = "git://github.com/${BPN}-ocr/${BPN}.git;branch=${BRANCH};protocol=https \
            file://0001-include-sys-time.h.patch \
           "
 
-- 
2.17.1

